exec/java-exec/src/main/java/org/apache/drill/exec/vector/complex/MapVector.java

```diff
@@ -309,7 +309,14 @@ public Object getObject(int index) {
       Map<String, Object> vv = new JsonStringHashMap<>();
       for (String child:getChildFieldNames()) {
         ValueVector v = getChild(child);
-        if (v != null) {
+        // TODO(DRILL-4001):  Resolve this hack:
```

Comment:
Added mention of MapVector.getObject(int) and exception message "java.lang.IndexOutOfBoundsException: index: 0, length: 1 (expected: range(0, 0))" to DRILL-4001 report.